scripts/importer/tests/data/reference.psql

```diff
@@ -1,4 +1,7 @@
-COPY beacon.beacon_dataset_counts_table (datasetid, dataset, callcount, variantcount) FROM stdin;
+COPY beacon.beacon_dataset_counts_table (datasetid, callcount, variantcount) FROM stdin;
```

Comment:
I will probably rewrite this file from scratch before I'm ready for merging this, so this part will probably have to be redone.